frontend/packages/ux-editor/src/containers/DesignView/FormLayoutWarning.module.css

```diff
@@ -0,0 +1,10 @@
+.warningWrapper {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
```

Comment:
Can we use some of the spacing tokens from the design system here? 😄

Reply:
Absolutely! 😄